com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -525,6 +528,20 @@ public static bool IsGameCamera(Camera camera)
             return camera.cameraType == CameraType.Game || camera.cameraType == CameraType.VR;
         }
 
+        /// <summary>
+        /// Checks if a camera is rendering in stereo mode.
+        /// </summary>
+        /// <param name="camera">Camera to check state from.</param>
+        /// <returns>Returns true if the given camera is rendering in stereo mode, false otherwise.</returns>
+        [Obsolete("Please use CameraData.xr.enabled instead.", true)]
+        public static bool IsStereoEnabled(Camera camera)
```

Comment:
LGTM. The way I understand you can just throw an excepcional here. But this is also fine.